english/LifePuzzle.md

```diff
@@ -0,0 +1 @@
+Life is full of small puzzles that we try to fit to make sense of it all.
```

Comment:
Hi,


I did make changes in my branch of the fork and I am able to see that this page is reachable from the rest of the story.I think I might need some help to understand what I did wrong.

Thanks,

Renuka



________________________________
From: Casey Collaborator <notifications@github.com>
Sent: Thursday, April 21, 2016 6:04 PM
To: udacity/create-your-own-adventure
Cc: renukaramkumar; Author
Subject: Re: [udacity/create-your-own-adventure] Create LifePuzzle (#4308)


In english/LifePuzzle.md<https://github.com/udacity/create-your-own-adventure/pull/4308#discussion_r60678587>:

> @@ -0,0 +1 @@
> +Life is full of small puzzles that we try to fit to make sense of it all.


It looks like this page is not reachable from the rest of the story. That is, if a reader starts in language.md and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/udacity/create-your-own-adventure/pull/4308/files/33c45f63e7199a3542d624b53309b810a77f2727#r60678587>